packages/extension-base/src/background/handlers/Extension.ts

```diff
@@ -477,7 +477,7 @@ export default class Extension {
 
   // this method is called when we want to open up the popup from the ui
   private windowOpen (path: AllowedPath): boolean {
-    const url = `${chrome.extension.getURL('index.html')}#${path}`;
+    const url = `${chrome.extension.getURL('external.html')}#${path}`;
```

Comment:
I thought `notification.html` is new window popup and `index.html` is an extension popup, isn't that correct?

Reply:
It wasn't like that - `index.html` was used for both extension popup and _some_ external windows. What I did was splitting this use case into 2 separate html files. So now we have one _isolated_ usage of the extension popup (`index.html`) and 2 types of external windows (the new `external.html` and the old, intouched `notification.html`).

Maybe I could've merged the 2 types of external windows, but the previous division acros _different axis of responsibilities_ makes me uncomfortable to do so without vast manualt testing that'd take much more time than this ticket deserves.